refinery/data_set_manager/models.py

```diff
@@ -515,13 +567,13 @@ def get_file_store_item(self):
         Returns the FileStoreItem associated with a given Node or None if
         there isn't one
         """
-        try:
-            return FileStoreItem.objects.get(
-                uuid=self.file_uuid)
-        except (FileStoreItem.DoesNotExist,
-                FileStoreItem.MultipleObjectsReturned) as e:
-            logger.error(e)
-            return None
+        if self.file_uuid:
```

Comment:
Many Nodes aren't file Nodes. and this logger.error was widespread throughout logs making things confusing